Kajabi/sage-lib
# Tooltip - add rails component and update docs

## Description

- [ ] - update docs and props

## Test notes

No visual chances

### Steps for testing
1. Visit tooltip sage rails page: http://localhost:4000/pages/element/tooltip
2. Verify demo and docs
